brainstormforce/ultimate-addons-for-gutenberg
# UAG-385 - Fix: UAG blocks losses style on single post when Auto Load Previous Posts is ON.

### Description

- https://ua-gutenberg.atlassian.net/browse/UAG-385

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Create few posts.
- Add the UAG block to the content of those posts.
- Activate "Auto Load Previous Posts" option in Customizer > Blog > Single Post.
- Click on full Post text and scroll down to the next auto-loaded post. You will see that the UAG block doesn't have styling. It doesn't look good. 
- Switch to `uagb-block-losses-style-when-auto-load-pre-post-is-on` Check UAG block asset is loading on the post and all is working fine.
- Disable the `Auto Load Previous Posts` and check again.
- Switch to a different theme and regenerate assets and check post everything looks good.

### Checklist:
